gora.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>WELCOME ON SPIRIT8</title>
+    <link rel="stylesheet" href="css/gora.css">
+	<link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@300&family=Cutive+Mono&family=Dancing+Script:wght@700;600&family=Darker+Grotesque:wght@800&family=Julius+Sans+One&family=Lato:ital,wght@1,300&family=Nanum+Gothic&family=Playfair+Display:ital@1&display=swap" rel="stylesheet">
+    
+  </head>
+<body>
+  <div class="overlay"></div>
+   <header class="header">
+    <div class="container">
+       <div class="header-inner">
+           <div class="header-logo">Spirit8</div>
+           <nav class="nav">
+               <a class="nav-link" href=".....">Home</a>
+               <a class="nav-link" href=".....">about</a>
+               <a class="nav-link" href=".....">team</a>
+               <a class="nav-link" href="......">services</a>
+               <a class="nav-link" href="......">porfolio</a>
+               <a class="nav-link" href="......">testimonials</a>
+               <a class="nav-link" href="......">contact</a>
+           </nav>
+       </div>
+     </div>
+ </header>
+    <div class="intro">
+        <div class="container">
+           <div class="intro-inner">
+                <p class="suptitle">welcome on <span class="color-text">spirit8</span></p>
+                <p class="intro-title">We are a digital agency with <b>years of experience</b> and with <b>extraordinary people</b></p>
+              <div class="button"><a class="arrow" href="#bot"></a></div>
+           </div>
+        </div> 
+    </div> 
+<!-- next page================================-->
+<div class="body-img">
+     <div class="textcols">
+       <img src="../gora/img/02.png" width="550px" height="500px" alt="computer" >
+     </div>
+     <div class="textcols">
+      <div class="subtitle">
+       <P class="text3">about us <br>some words <b>about us</b></P>
+     	<div class="border-bottom"></div>
+      </div>
+    	  <span>We love building and rebuilding brands through our  specific skills. Using colour, fonts, and illustration, we brand companies in a way they will never forget.</span>
+      <div class="nav2">
+        <ul class="nav-intro2">
+    	  <li ><a class="nav-link2" href="#"><b>Mission</b> - <em>we deliver uniqueness and quality</em></a></li>
+    	  <li><a class="nav-link2"href="#"><b>Skills</b> - <em>delivering fast and excellent results</em></a></li>
+    	  <li><a class="nav-link2" href="#"><b>Clients</b> -<em> satisfied clients thanks to our experience</em></a></li>
+        </ul>
+      </div>
+    </div>
+</div>
+ <!-- next page================================-->
+<div class="bg">
+  <div class="personal">
+ 	<div class="sub-text">
+ 	    <div>
```

Comment:
add 
```css
.inform_header {
    display: flex;
    flex-direction: column;
    align-items: center;
}
```
 class
put `<div class="border-intro"></div>` inside the `inform_header `
remove `div class="personal">
 	<div class="sub-text">`
 	
 	
```css
.bg {
	background: url(../img/03.jpg) no-repeat center;
	background-size:cover;
	justify-content: center;
	flex-direction: column;
	display: flex;
}
```

remove from classes for lines `displat: flex-inline`